tesoro/patch.py

```diff
@@ -39,6 +40,19 @@ def make_patch(src_json, dst_json):
     for idx, patch_item in enumerate(patch):
         if patch_item["path"] == last_applied:
             patch.pop(idx)
-            logger.debug("Removed last-applied-configuration annotation" "from patch")
+            logger.debug("Removed last-applied-configuration annotation from patch")
 
     return patch
+
+
+def redact_patch(patch):
+    "returns a copy of patch with redacted values"
+    redacted_patch = deepcopy(patch)
+
+    for patch_item in redacted_patch:
+        # don't redact this annotation
+        if patch_item["path"] == "/metadata/annotations/tesoro.kapicorp.com~1revealed":
+            continue
+        patch_item["value"] = "!REDACTED VALUE!"
```

Comment:
I'm thinking that this wouldn't really help us debug the current issue we have. How about a hash of the secret instead? 
```
patch_item["value"] = "!REDACTED VALUE! digest={}".format(hashlib.sha256(patch_item["value"]).hexdigest())
```

or something? this way at least we can understand what's going on

Reply:
Yep agree. Will push fix soon